cogs/helpers.py

```diff
@@ -148,14 +148,19 @@ def __init__(self, bot):
         tendency_label = soup.find("dt", string="Tendency:")
         # Get wind
         wind_label = soup.find("dt", string="Wind:")
-        # Get windchill, only if it can be found.
-
-        try:
-            windchill_label = soup.find("a", string="Wind Chill")
-            windchill = windchill_label.find_next().get_text().strip(
-            ) + u"\xb0C"
-        except Exception:
-            windchill = u"N/A"
+        # Get windchill using standard formula from Environment Canada
+        temperature = float(
+            re.search(r"-?\d+\.\d",
+                      temperature_label.find_next_sibling().get_text().strip()
+                      ).group())
+        wind_speed = float(
+            re.search(
+                r"\d+",
+                wind_label.find_next_sibling().get_text().strip()).group())
```

Comment:
Since you're parsing the HTML here to get temperature and wind speed, you can pass these variables to the embed below instead of those values being filled by parsing again. You might have to cast them as `str` though, so their new values would be something like `value=str(temperature)` and `value=str(wind_speed)` when constructing the `Embed` object.

Reply:
They aren't parsed the same way though. For example, the string that is sent to embed is "WSW 68 gust 93 km/h", while in my wind_speed variable it is regexed to get only 68

Reply:
In that case then I suggest creating a new variable just to store the wind velocity string, and then you can regex that string just for the wind speed and the wind velocity string can be passed to the Embed object. This will reduce the number of parsing operations to get the same string. The wind velocity string is needed for both values: 1) it is regex-filtered for the windchill calculation, and 2) it is passed directly to the Embed.

The same thing could be done for temperature too.